src/NuGet.Services.EndToEnd/PushTests.cs

```diff
@@ -106,5 +108,30 @@ public async Task NewlyPushedIsODataSearchableInDB(PackageType packageType)
             await _clients.FlatContainer.WaitForPackageAsync(package.Id, package.NormalizedVersion, _logger);
             await _clients.Gallery.SearchPackageODataV2FromDBAsync(package.Id, package.NormalizedVersion, _logger);
         }
+
+        /// <summary>
+        /// Push a package to the gallery and verify the owner information is propagated.
+        /// </summary>
+        [AzureSearchOnly]
+        public async Task NewlyPushedPackageHasOwnerInformation()
+        {
+            // Arrange
+            var package = await _pushedPackages.PrepareAsync(PackageType.SemVer1Stable, _logger);
+            var searchServices = await _clients.V2V3Search.GetSearchServicesAsync(_logger);
+
+            // wait for package to become available
+            await _clients.V2V3Search.WaitForPackageAsync(package.Id, package.FullVersion, _logger);
+
+            // Act
+            foreach (var searchService in searchServices)
+            {
+                var result = await _clients.V2V3Search.QueryAsync(searchService, $"q=packageid:{package.Id} owner:{_testSettings.TestAccountOwner}", _logger);
+                Assert.NotNull(result);
+                Assert.NotNull(result.Data);
+                Assert.NotEmpty(result.Data);
```

Comment:
`NotEmpty` and the count assertion are redundant. I recommend just removing this one.